Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM frolvlad/alpine-nim
+FROM nimlang/nim:2.0.4-alpine-slim
 
-RUN apk update
-RUN apk add bash git gmp-dev mpfr-dev pcre-dev sqlite-dev 
+RUN apk add --no-cache bash git gmp-dev gtk+3.0-dev mpfr-dev pcre-dev sqlite-dev webkit2gtk-dev
```

Comment:
In preparation for a multi-stage build: which of these dependencies are also runtime dependencies? In other words: which of these dependencies are only needed for compilation, and which are required to run things? 

I can imagine `pcre-dev` being needed for compilation, but could the regular `pcre` image also be used once it is compiled?